### DISCUSSION TITLE
Broken links to prowler-additions-policy.json and prowler-security-hub.json

### DISCUSSION THREAD

**Discussion by pcuttriss** [prowler-cloud/prowler#1682] [General]
Afternoon all

Documentation observation: The Documentation/AWS section lists the two files in the title but the location appears to have changed in the repo.
Original location: https://github.com/prowler-cloud/prowler/blob/master/iam/prowler-additions-policy.json
Current location:  https://github.com/prowler-cloud/prowler/blob/master/permissions/prowler-additions-policy.json

Looks like there has been a recent move for https://github.com/prowler-cloud/prowler/pull/1531

Ta
Paul C

**Comment by toniblyx** [comment]
Thanks @pcuttriss I´m going to create an issue with this to fix it.

**Reply by pcuttriss** [reply]
Thanks for all of your work on an excellent tool.....it's keeping me very occupied at work these days
